core/cron/pokedex.rarity.php

```diff
@@ -1,9 +1,5 @@
 <?php
 
-
-// This file is loaded once every 24h by data.loader.php.
-// -----------------------------------------------------
-
 // get alltime pokemon count to set rarity at least to seen
 $req = "SELECT pokemon_id, COUNT(*) as spawns_total FROM pokemon GROUP BY pokemon_id ORDER BY pokemon_id ASC";
```

Comment:
We are using innoDB and it should be better to do:
COUNT(encounter_id) or COUNT(1) then COUNT(*)
http://stackoverflow.com/questions/2710621/count-vs-count1-vs-countpk-which-is-better